lua/lint/linters/eslint.lua

```diff
@@ -1,27 +1,74 @@
-local pattern = [[%s*(%d+):(%d+)%s+(%w+)%s+(.+%S)%s+(%S+)]]
-local groups = { 'lnum', 'col', 'severity', 'message', 'code' }
-local severity_map = {
-  ['error'] = vim.diagnostic.severity.ERROR,
-  ['warn'] = vim.diagnostic.severity.WARN,
-  ['warning'] = vim.diagnostic.severity.WARN,
+local binary_name = "eslint"
```

Comment:
I added this since the `eslint` and `eslint_d` linters are mirrors. We could set this up differently to keep things more DRY, but yeah... 🤷🏼